src/components/seeds/summary/index.tsx

```diff
@@ -189,7 +189,7 @@ export default function SeedSummary(props: SeedSummaryProps): JSX.Element {
                     <Typography fontSize='20px' color='#000000' paddingBottom='8px' paddingLeft='28px'>
                       {strings.ACCESSION_BY_STATUS}
                     </Typography>
-                    <Link className={classes.accessionsLink} to={APP_PATHS.ACCESSIONS}>
+                    <Link className={classes.accessionsLink} to={`${APP_PATHS.ACCESSIONS}?stage=}`}>
```

Comment:
Why is this necessary?

Reply:
We need a way to say load all stages (since there might be an existing filter). Using `stage=` for that. Thought about `stage=*` but `stage=` would clear previous filters, and `stage=<something` would set the next filter. Open to suggestions.

Reply:
I think this is ok for now. I didn't realize this wasn't working more than 1 time

Reply:
Thanks. Yes I didn’t realize that either. I’ll update the web-components version too and update this PR.